op-node/rollup/driver/state.go

```diff
@@ -168,8 +170,7 @@ func (s *Driver) eventLoop() {
 	defer s.wg.Done()
 	s.log.Info("State loop started")
 
```

Comment:
Wondering if we want to take the opportunity to add logging for when we return from the loop
```suggestion
defer s.log.Info("State loop returned")
```

Reply:
https://github.com/ethereum-optimism/optimism/pull/8295